### PR TITLE
Add run on js docs note

### PR DESCRIPTION
**Pull request by karol-bisztyga**
## Description

Add run on js docs note

### CODE REVIEW

File: docs/docs/api/runonJS.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+id: runOnJS
+title: runOnJS
+sidebar_label: runOnJS
+---
+
+When you call a function on UI thread you can't be sure if you call a worklet or a callback from JS thread. To make it more transparent we introduce runOnJS method which calls a callback asynchronously. If you call a callback without this method then an exception will be thrown.
+
+#### Note
```
Comment (jakub-gonet) [inline docs/docs/api/runonJS.md:9]: I think this should be a quote block (do we have any guidelines on docs structure?)

I'd add a short explanation of why we can't wrap the function directly.

Comment (jakub-gonet) [reply docs/docs/api/runonJS.md:9]: Anyway, maybe we should consider wrapping functions automatically?

File: docs/docs/api/runonJS.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
+id: runOnJS
+title: runOnJS
+sidebar_label: runOnJS
+---
+
+When you call a function on UI thread you can't be sure if you call a worklet or a callback from JS thread. To make it more transparent we introduce runOnJS method which calls a callback asynchronously. If you call a callback without this method then an exception will be thrown.
```
[the remark this answers is not in the document]
Comment (karol-bisztyga) [reply docs/docs/api/runonJS.md:7]: > "method" -> function

look like repetition, we can just skip the word. The rest is ok

File: docs/docs/api/runonJS.md (hunk 1 above)
Comment (karol-bisztyga) [reply docs/docs/api/runonJS.md]: fixed fd4e410

**Review by jakub-gonet — APPROVED** [review]

**Pull request merged**